### PR TITLE
[REF] add hook to ease overriding the merge of automatic po

### PR DESCRIPTION
**Pull request by florian-dacosta**
In some case we do not want to merge purchase orders. It would be nice to have a way to custom that easily.

Thanks.

### CODE REVIEW

**Comment by yvaucher** [comment]
@florian-dacosta needs a rebase

**Comment by florian-dacosta** [comment]
@yvaucher done!

**Comment by nim-odoo** [comment]
Hello,
Thanks for the PR. Could we use a private method instead? I don't see the benefit of a public method here.

**Comment by nim-odoo** [comment]
Hello,
Without any feedback, I close the PR. Please reopen if necessary.
Regards

File: addons/purchase/purchase.py [hunk 1]
```diff
@@ -1373,6 +1373,23 @@ def _calc_new_qty_price(self, cr, uid, procurement, po_line=None, cancel=False,
             price = pricelist_obj.price_get(cr, uid, [pricelist_id], procurement.product_id.id, qty, po_line.order_id.partner_id.id, {'uom': procurement.product_uom.id})[pricelist_id]
 
         return qty, price
+  
+    def get_available_draft_po_ids(self, cr, uid, procurement, partner,
```
Comment (yvaucher) [inline addons/purchase/purchase.py]: @florian-dacosta Can you add a little `_` here ?

**Pull request closed without merging**